### ISSUE TITLE
MUDS dataset

### ISSUE THREAD

**Issue by HQA01** [hiyamdebary/EarthDial#9]
Thank you for your excellent work. I would like to inquire about the size of the MUDS dataset after your processing. I only found 250 training sets and 10 test sets in the link you provided, and I also couldn't find the corresponding masks for the change areas.

**Comment by mustansarfiaz** [comment]
Hi  
I appreciate your interest. 
You are right, during the data curation time, the MUDS [1] paper released 60 examples. So, we used 50 examples as the training set and 10 examples as the test set.

1. Yang, Charig, Weidi Xie, and Andrew Zisserman. "Made to order: Discovering monotonic temporal changes via self-supervised video ordering." European Conference on Computer Vision. Cham: Springer Nature Switzerland, 2024.

Regards
Mustansar

**Comment by HQA01** [comment]
Okay, thank you for your prompt reply.

> Hi I appreciate your interest. You are right, during the data curation time, the MUDS [1] paper released 60 examples. So, we used 50 examples as the training set and 10 examples as the test set.
> 
> 1. Yang, Charig, Weidi Xie, and Andrew Zisserman. "Made to order: Discovering monotonic temporal changes via self-supervised video ordering." European Conference on Computer Vision. Cham: Springer Nature Switzerland, 2024.
> 
> Regards Mustansar

Okay, thank you for your prompt reply
